e2e/docker-compose.yml

```diff
@@ -78,6 +78,8 @@ services:
       POSTGRES_USER: app_user_dev
       POSTGRES_PASSWORD: DevPassword
       POSTGRES_DB: app_dev_db
+    ports:
+      - 127.0.0.1:5432:5432
```

Comment:
[nit] Maybe this should be bound to some other port number on the host, in case someone wanted to run dev & e2e stacks at the same time?